__tests__/app.test.js

```diff
@@ -117,3 +117,52 @@ describe("GET /reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /reviews/:review_id/comments", () => {
+  test("GET - status:200, responds with a review object", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(3);

```

Comment:
Add a check in here somewhere that checks that the comments are coming back in the correct order - newest first.